code/modules/mob/mob_movement.dm

```diff
@@ -1,5 +1,5 @@
 /mob/CanPass(atom/movable/mover, turf/target, height=1.5, air_group = 0)
-	if(air_group || (height==0)) return 1
+	if(air_group || (height==0) || (incorporeal_move == INCORPOREAL_ETHEREAL)) return 1

```

Comment:
how would you code around that? I don't see it, without adding more snowflake in the "get up" code.